Functions.Templates/ProjectTemplate/CSharp-Isolated/.template.config/template.json

```diff
@@ -0,0 +1,56 @@
+{
+    "author": "Microsoft",
+    "classifications": [
+        "Azure Functions",
+        "Solution"
+    ],
+    "name": "Azure Functions",
+    "generatorVersions": "[1.0.0.0-*)",
+    "groupIdentity": "Microsoft.AzureFunctions.ProjectTemplates",
+    "precedence": "100",
+    "identity": "Microsoft.AzureFunctions.ProjectTemplate.CSharp.4.x",
+    "shortName": "func",
+    "tags": {
+        "language": "C#",
+        "type": "project"
+    },
+    "sourceName": "Company.FunctionApp",
+    "symbols": {
+        "Namespace": {
+            "description": "namespace for the generated code",
+            "replaces": "Company.FunctionApp",
+            "defaultValue": "FunctionApp",
```

Comment:
Need to verify if we can replace `Company.FunctionApp` with directory name. This is for the namespace in `Program.cs`.

Reply:
Use this file as an example. https://github.com/Azure/azure-functions-templates/blob/dev/Functions.Templates/Templates/CosmosDBTrigger-CSharp/.template.config/template.json. Match the sourceName in respective locations. It should be same as the class file name. Cannot keep class file name as program.cs otherwise all `.cs` files for function would end up being named program.cs